staging/src/k8s.io/code-generator/cmd/register-gen/generators/packages.go

```diff
@@ -82,7 +82,7 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 
 			// if there is a comment of the form "// +groupName=somegroup" or "// +groupName=somegroup.foo.bar.io",
 			// extract the fully qualified API group name from it and overwrite the group inferred from the package path
-			if override := types.ExtractCommentTags("+", pkg.DocComments)["groupName"]; override != nil {
+			if override := types.ExtractCommentTags("+", pkg.Comments)["groupName"]; override != nil {
```

Comment:
does Comments contain all comments? Note, we have this strange distinction between comments directly above something and those separated by an empty line.

Reply:
Ic, Comments means all:
```
// All comments from doc.go, if any.
// TODO: remove Comments and use DocComments everywhere.
Comments []string
```

Do we have other places like that?

Reply:
As far as I see the other 6 code snippets accessing groupName are fine, just this one.

Reply:
yes, I fount this bug  just this code . other code is right.